src/hooks/use-combined-refs/use-combined-refs.stories.mdx

```diff
@@ -0,0 +1,21 @@
+import { Meta } from '@storybook/addon-docs'
+import { useCombinedRefs } from './use-combined-refs'
+
+<Meta title="Hooks/useCombinedRefs" component={useCombinedRefs} />
+
+# useCombinedRefs
+
+`useCombinedRefs` allows you to merge a passed down from above and your own ref together into one reference.
+
+```TypeScript
+import { useCombinedRefs } from '@doist/reactist'
+
+function MyComponent(props, ref) {
+    const myRef = useRef()
+    const combinedRef = useCombinedRefs(ref, myRef)
+
+    return (
+        <Box ref={combinedRef} />
+    )
+}
+```
```

Comment:
I suspect that the functionality provided by this hook must be already offered by the generic hooks library we use. So my point is: do we really want to export this from Reactist? It forces us to either maintain it in the future, or go through a cumbersome deprecation cycle if we decide to remove it. Especially now that we no longer offer any hooks from within Reactist.

Reply:
Good call, [useMergeRefs](https://github.com/theKashey/use-callback-ref#usemergerefs) works identically here 👍